sass/px-datetime-range-panel.scss

```diff
@@ -19,43 +21,37 @@
 @import "px-shadows-design/_objects.shadows.scss";
 
 // Component
-:host {
-  text-align: left;
-}
 :host[hidden] {
   visibility: hidden;
 }
-.btn--bare{
-  &:hover, &:focus {
-    outline: none;
-  }
-}
+
 .rangepicker__box {
   position: absolute;
+  padding: calculateRem(10px) calculateRem(20px) calculateRem(20px) calculateRem(20px);
   border: 1px solid;
-  z-index:2;
+  z-index: var(--px-range-panel-z-index, 300);
```

Comment:
We are trying to be more consistent with our z-indexing using this as guide https://ge.ent.box.com/notes/161776002634